lib/full-site-editing/edit-site-page.php

```diff
@@ -31,6 +31,15 @@ function gutenberg_is_edit_site_page( $page ) {
 	return 'appearance_page_gutenberg-edit-site' === $page;
 }
 
+// Default to is-fullscreen-mode to avoid rendering wp-admin navigation menu while loading and
+// having jumps in the UI.
+add_filter(
```

Comment:
This is going to be executed for all pages, not just the site editor (because the php file is always loaded), should we instead move this into `gutenberg_edit_site_init` ?

Reply:
Ohh, that makes sense.  I moved it into that function here c434238

Reply:
Do we need to also remove the filter as part of the `! gutenberg_is_edit_site_page( $hook )` conditional return?

Reply:
Also curious, does the function need to be `static` here?

Reply:
they started using "static" for closures like that recently something about PHP 8 support, but I didn't fully follow the explanations tbh :) So I might think it's better if it's static here as well.

It doesn't seem like we need to remove the filter though as it's only applied if we're on this page so, so it looks good to me.